content/en/docs/project-user-guide/application-workloads/ingress.md

```diff
@@ -1,10 +1,135 @@
+  - **https**: In addition to the configuration contained in http, a secret is also required. The secret can be specified to contain a TLS private key and certificate to encrypt Ingress, and the TLS secret must contain keys named tls.crt and tls.key.
+
+![](/images/docs/route-set-rule-domain.png)
+
+### Step 4: Advanced Settings
+
+You can edit the metadata info of the route in Advanced Settings.
+
+Ingress provides annotations to customize behavior. Please refer to [Ingress Annotations](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/).
+
+![](/images/docs/route-create-annotations.png)
+
+## Access the Route
+
+After the route is created, make sure that the domain name can be resolved to the IP address of the external network access entrance, and you can use the domain name access. For example, in a private environment, you can modify the local hosts file to use the route. E.g:
+
+|Domain|Path|Access Type|Port/IP|Cluster Node IP|
+----|---|---|---|---
+|demo.kubesphere.io|/|NodePort|32586,31920|192.168.0.4,192.168.0.3,192.168.0.2|
+|demo2.kubesphere.io|/|LoadBalancer|139.198.1.1|192.168.0.4,192.168.0.3,192.168.0.2
+
+As shown in the above table, two route rules have been created, using NodePort and LoadBalancer access entries respectively.
+
+- **NodePort**
+
+    For services with internet access set to NodePort, if it is in a private environment, you can directly add records in the hosts file of the host to resolve the domain name to the corresponding IP. For example, for `demo.kubesphere.io`, adding the following records:
+
+    ```
+    192.168.0.4 demo.kubesphere.io
+    ```
+
+    It is necessary to ensure that the client and the cluster working node `192.168.0.4` network can communicate, and the IP of other working nodes can be used. As long as the client and the working node network are connected, after setting, use `http://demo.kubesphere.io:32586` to access the backend service.
+
+- **LoadBalancer**
+
+    For services with internet access set to LoadBalancer, for `demo2.kubesphere.io`, in addition to adding records in the hosts file by referring to the **NodePort**, you can also use `nip.io` as the domain name resolution for routing. `nip.io` is a free domain name resolution service, which can resolve the corresponding ip of the domain name in the following format, which can be used as a resolution service for routing, eliminating the need to configure the local hosts file.
+
+    ```
+    10.0.0.1.nip.io maps to 10.0.0.1  
+    app.10.0.0.1.nip.io maps to 10.0.0.1
+    customer1.app.10.0.0.1.nip.io maps to 10.0.0.1
+    customer2.app.10.0.0.1.nip.io maps to 10.0.0.1
+    otherapp.10.0.0.1.nip.io maps to 10.0.0.1
+    ```
+
+## Check Route detail
+
+### Route Operations
+
+- **Edit Info**: Editing the basic information except `Name` of the Route, .
```

Comment:
Editing the basic information except `Name` of the Route, .
->
Edit the basic information except `Name` of the Route.